### ISSUE TITLE
parallel train

### ISSUE THREAD

**Issue by Tony-Hou** [hustvl/YOLOP#61]

**Comment by Tony-Hou** [comment]
How to parallel train with pytorch?

**Comment by SikandAlex** [comment]
https://pytorch.org/tutorials/beginner/dist_overview.html#data-parallel-training

Follow this document. You will likely need SyncBN instead of standard BatchNorm. The authors trained the network using a single Titan Xp and I trained the network using a single RTX 2080 Ti.
